modules/network-observability-resources-table.adoc

```diff
@@ -0,0 +1,28 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+:_content-type: REFERENCE
+[id="network-observability-resources-table{context}"]
+= Resource recommendations
+The following table documents recommendations for resources depending on workload size. 
+
+[IMPORTANT]
+====
+These recommended scenarios are tailored to specific workloads and should serve only as a baseline from which adjustments can be made to accommodate your workload needs.
+====
+
+.Resource recommendations
+[options="header"]
+|============================================================================================================================
+|                                     | Extra small (10 Nodes) | Small (25 nodes)  | Medium (65 Nodes) ^[2]^ | Large (120 nodes) ^[2]^
+| *Worker Node vCPU and Memory*       | 4 vCPUs\| 16GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem ^[1]^ | 16 vCPUs\| 64GiB Mem  ^[1]^  |16 vCPUs\| 64GiB Mem ^[1]^ 
+| *LokiStack Size*                    | 1x.extra-small         | 1x.small          | 1x.small           | 1x.medium
+| *NetObserv Controller Memory Limit* | 400Mi (default)        | 400Mi (default)   | 400Mi (default)    | 800Mi                
+| *eBPF Sampling Rate*                | 50 (default)           | 50 (default)      | 50 (default)       | 50 (default)
+| *eBPF Memory Limit*                 | 800Mi (default)        | 800Mi (default)   | 2000Mi             | 800Mi (default) 
+| *FLP Mem Limit*                     | 800Mi (default)        | 800Mi (default)   | 800Mi (default)    | 800Mi (default)         
+| *FLP Kafka Partitions*              | N/A                    | 48                | 48                 | 48            
+| *Kafka Consumer Replicas*           | N/A                    | 24                | 24                 | 24
+| *Kafka Brokers*                     | N/A                    | 3 (default)       | 3 (default)        | 3 (default)
+|============================================================================================================================
+1. Tested with AWS m6i instances.
```

Comment:
M6i? as per https://aws.amazon.com/about-aws/whats-new/2021/08/amazon-ec2-m6i-instances/

Reply:
Ah yep, that's right.